doc/classes/Physics2DServer.xml

```diff
@@ -4,7 +4,91 @@
 		Server interface for low-level 2D physics access.
 	</brief_description>
 	<description>
-		Physics2DServer is the server responsible for all 2D physics. It can create many kinds of physics objects, but does not insert them on the node tree.
+		Physics2DServer is the server responsible for 2D physics. Physics2DServer exposes static methods that manipulate resources directly, bypassing [Node2D] and the system of classes that inherit from it. As a consequence, resources created by Physics2DServer are not attached to the node tree. Developers should reluctantly bypass the higher level of abstraction provided by Godot’s 2D API, but Physics2DServer is nonetheless sometimes unavoidable.
+		A basic understanding of Physics2DServer’s primitive types is necessary to use the class successfully. Take care not to confuse Physics2DServer’s primitive types with their [Node2D] counterparts.
+		A [code]space[/code] is a physics environment in which an [code]area[/code] or [code]body[/code] can be placed. Once attached to the same space, areas and bodies are tested for collision. Typically, the default space instantiated by Godot’s scene tree is sufficient but multiple spaces can run in parallel.
+		Anything that collides in 2D physics is either a [code]body[/code] or an [code]area[/code]. Area is the same as the resource used by an Area2D object. Body is the same as the resource used by a RigidBody2D object.
+		A [code]shape[/code] gives physical extension to [code]area[/code] and [code]body[/code] for the purpose of collision in physical [code]space[/code]. Shapes are the basis of Shape2D.
+		A [code]joint[/code] modifies the behaviour of two bodies based on their relationship to each other. They are the same as the resources used by Joint2D.
+		Godot assigns an [code]object_instance_id[/code] to a Node2D when it is instantiated. This id is unique for each object in the node tree. Because Physics2DServer does not instantiate classes as [code]Node2D[/code] objects, these resources do not have valid object instance IDs and are not suitable for attachment to the node tree. Instead, they default to zero.
+		As with object instance IDs, Physics2DServer does not create resources with a valid [code]canvas_instance_id[/code]. Instead, the canvas ID will be zero. The canvas is the same as is used by a [code]Canvaslayer[/code].
+		There is currently no supported method for casting areas or bodies created with Physics2DServer as Node2D objects. Because of this, you’re better off instantiating nodes from Node2D.new() or one of Node2D’s member classes, then using get_rid() to retrieve an RID Physics2DServer can work with.
+		Attach the following script to a Node2D in a new Godot project and run it. It demonstrates many of the basic relationships between Physics2DServer functions.
+	[codeblock]
+	extends Node2D
+
+	var aFooBody:RID
+	var aBarBody:RID
+	var gravityWell:RID
```

Comment:
We shouldn't put such giant codeblocks in the docs. It also needs a C# counterpart. From the looks of it, this description can be made into a dedicated tutorial article and just linked here.